InternxtDesktop/Views/Widget/WidgetFolderList.swift

```diff
@@ -10,12 +10,11 @@ import SwiftUI
 struct WidgetFolderList: View {
```

Comment:
The View is not placed correctly, please name this something related to Backups and move it inside `Views/Backups/**`, `BackupsFolderList` would be a better name